test/metabase/search/fulltext/scoring_test.clj

```diff
@@ -79,9 +79,40 @@
               ["card" 1 "card ancient"]]
              (search :recency "card"))))))
 
+(deftest ^:parallel view-count-test
+  (testing "the more view count the better"
+    (with-index-contents
+      [{:model "card" :id 1 :name "card famous"     :view_count 100}
+       {:model "card" :id 2 :name "card popular"    :view_count 50}
+       {:model "card" :id 3 :name "card well known" :view_count 10}]
+      (is (= [["card" 1 "card famous"]
+              ["card" 2 "card popular"]
+              ["card" 3 "card well known"]]
+             (search :view-count "card")))))
+  (testing "don't error on fresh instances with no view count"
```

Comment:
You'll need to bust the cache so that we recalculate the percentiles

Reply:
I've made the cache prod-only, unfortunately, if we keep the catch during its it'll affects other tests as well.